get_data_for_figures.py

```diff
@@ -187,44 +210,86 @@ def main(args):
             popmax=ht_indexed.popmax,
         )
 
-        # Filter to only variants with a popmax allele frequency of 0.1%
-        mt = mt.filter_rows(mt.popmax[0].AF < 0.001)
+        logger.info("Reading in v2 genomes, v3 genomes, and v2 liftover tables.")
+        v2_genomes = hl.read_table(
+            "gs://gcp-public-data--gnomad/release/2.1.1/ht/genomes/gnomad.genomes.r2.1.1.sites.ht"
+        )
+        v3_genomes = hl.read_table(
+            "gs://gcp-public-data--gnomad/release/3.1.1/ht/genomes/gnomad.genomes.v3.1.1.sites.ht"
+        )
+        v2_liftover = hl.read_table(
+            "gs://gcp-public-data--gnomad/release/2.1.1/liftover_grch38/ht/exomes/gnomad.exomes.r2.1.1.sites.liftover_grch38.ht"
+        )
+        v2_liftover = v2_liftover.key_by("original_locus", "original_alleles")
+        # annotate liftover locus onto MT
+        v2_liftover_index = v2_liftover[mt.row_key]
+        mt = mt.annotate_rows(
+            liftover_locus=v2_liftover_index.locus,
+            liftover_allele=v2_liftover_index.alleles,
+        )
+        # get v2 exome variants from v2 genomes and v3 genomes, and annotate their respective AF
+        v2_genomes_indexed = v2_genomes[mt.row_key]
+        v3_genomes_indexed = v3_genomes[mt.liftover_locus, mt.liftover_allele]
+        mt = mt.annotate_rows(
+            v2_genomes_popmax=v2_genomes_indexed.popmax,
+            v3_genomes_popmax=v3_genomes_indexed.popmax,
+        )
+        logger.info(
+            "Filtering to variants with a popmax allele frequency of less than .001 in v2_exomes, AND v2_genomes, AND v3_genomes..."
+        )
+        # Filter to only variants with a popmax allele frequency of < 0.1% in v2_exomes, AND v2_genomes, AND v3_genomes
+        mt = mt.filter_rows(
+            ((mt.popmax[0].AF < 0.001) | hl.is_missing(mt.popmax[0].AF))
+            & (
+                (mt.v2_genomes_popmax[0].AF < 0.001)
+                | hl.is_missing(mt.v2_genomes_popmax[0].AF)
+            )
+            & (
+                (mt.v3_genomes_popmax.AF < 0.001)
+                | hl.is_missing(mt.v3_genomes_popmax.AF)
+            )
+        )
```

Comment:
If the popmax's are not used again I would change this to
```suggestion
        logger.info(
            "Filtering to variants with a popmax allele frequency of less than .001 in v2_exomes, AND v2_genomes, AND v3_genomes..."
        )
        mt = mt.filter_rows(
            hl.or_else(mt.popmax[0].AF < 0.001, True)
            & hl.or_else(mt.v2_genomes_popmax[0].AF < 0.001, True)
            & hl.or_else(mt.v3_genomes_popmax.AF < 0.001, True)
        )
```

Reply:
Also add to the logger about how missing is handled, or add a comment about it

Reply:
I agree that we probably don't need the popmax info from v2 genomes or v3 genomes - do you mean then: 
```
mt = mt.filter_rows(
            hl.or_else(mt.popmax[0].AF < 0.001, True)
            & hl.or_else(v2_genomes_indexed.popmax[0].AF < 0.001, True)
            & hl.or_else(v3_genomes_indexed.popmax.AF < 0.001, True)
        )
```

?

Reply:
oh yes, sorry about that I forgot to change the names, edited with that change now